demo/Saritasa.NetForge.Demo.Net7/Commands/Seed.cs

```diff
@@ -0,0 +1,183 @@
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.Reflection;
+using McMaster.Extensions.CommandLineUtils;
+using Saritasa.NetForge.Demo.Net7.Infrastructure.Seeders;
+
+namespace Saritasa.NetForge.Demo.Net7.Commands;
+
+/// <summary>
+/// Seed command. It is used to run seeders in Infrastructure.Seeders namespace. It calls
+/// the Seed method of seeder with arguments that user passed thru command line. Required arguments are
+/// Name (class name) and Count.
+/// </summary>
+[HelpOption]
+[Command("seed", Description = "Seed database with fake data.",
+    UnrecognizedArgumentHandling = UnrecognizedArgumentHandling.CollectAndContinue)]
+public class Seed
+{
+    private const string SeederNamespace = "Saritasa.NetForge.Demo.Net7.Infrastructure.Seeders";
+    private const string SeedMethodName = "Seed";
+
+    private readonly ILogger<Seed> logger;
+    private readonly IServiceProvider serviceProvider;
+
+    /// <summary>
+    /// The name (class) of the seeder.
+    /// </summary>
+    [Option("--name", Description = "Name of seeder.")]
+    [Required]
+    public required string Name { get; init; }
+
+    /// <summary>
+    /// Number of objects to generate.
+    /// </summary>
+    [Option("--count", Description = "Number of objects to generate.")]
+    public int Count { get; } = 200;
+
+    /// <summary>
+    /// Remaining not parsed arguments.
+    /// </summary>
+    public string[]? RemainingArguments { get; }
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="Seed" /> class.
+    /// </summary>
+    /// <param name="logger">Logger.</param>
+    /// <param name="serviceProvider">Service provider.</param>
+    public Seed(ILogger<Seed> logger, IServiceProvider serviceProvider)
+    {
+        this.logger = logger;
+        this.serviceProvider = serviceProvider;
+    }
+
+    /// <summary>
+    /// Command line application execution callback.
+    /// </summary>
+    public async Task OnExecuteAsync()
+    {
+        if (Count < 1)
+        {
+            logger.LogInformation("Nothing to generate.");
+            return;
+        }
+
+        // Run default seeder if no specific.
+        var classType = !string.IsNullOrEmpty(Name) ?
+            Type.GetType($"{SeederNamespace}.{Name}Seeder", false, true) :
+            typeof(AddressesSeeder);
```

Comment:
I think this version is more readable
```suggestion
        var classType = !string.IsNullOrEmpty(Name) 
            ? Type.GetType($"{SeederNamespace}.{Name}Seeder", false, true) 
            : typeof(AddressesSeeder);
```